index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+	<script src="https://kit.fontawesome.com/27ebd9ab14.js" crossorigin="anonymous"></script>
+</head>
+<body>
+	<header>
+	<nav>
```

Comment:
- [ ] As required in the  previous review you should start using the `nav` tag after your `a` tag but I noticed that you even put your logo in the nav tag, kindly remove your logo in your `nav` tag as required in the [previous review](https://github.com/Gardimy/Mobile-first/pull/1#discussion_r1145188707) :wink:

Reply:
Why do I have to remove the logo in the nav tag, I can understand that you have your style of coding, but I use best practices. Is it a requirement to follow your coding style???